### DISCUSSION TITLE
Macedonian language new

### DISCUSSION THREAD

**Discussion by spasemice** [home-assistant/frontend#15316] [Localization]
English Name: Macedonian
Native Name: Македонски

mk-MK

**Comment by Beta-7** [comment]
+1. Would love to contribute

**Comment by Shume00** [comment]
I would also like to contribute. I have a couple of friends that can help too. Please add the language.

**Comment by lageant** [comment]
I also want to contribute. Please add Macedonian language to Home Assistant.

**Comment by jstosic** [comment]
I would like to contribute. It would be nice to have Macedonian language in this excellent project.

**Comment by glisha** [comment]
Created the pull request https://github.com/home-assistant/frontend/pull/20701.

Was following https://developers.home-assistant.io/docs/translations/#maintainer-steps-to-add-a-new-language

Steps 1 and 2 from there will be done with this pull request being merged.

- It has the "mk" code in the IANA subtag registry.
- Adding it in src/translations/translationMetadata.json

**The third step adding it in Lokalize needs to be done by a maintainer, after that we can start contributing.**
